### PR TITLE
Improve init sync performance

### PR DESCRIPTION
**Pull request by bmarty**
This PR reduce the delay to see the room list and be able to open a room and send a message to it.
On my account (about 300 rooms) I can see the room list after 55 seconds. With this PR, it is reduced to 35 seconds, including 12 seconds to let the server compute the response and the client to download the data.

### CODE REVIEW

File: matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/sync/FilterService.kt [hunk 1]
```diff
@@ -22,9 +22,9 @@ interface FilterService {
         NoFilter,
 
         /**
-         * Filter for Riot, will include only known event type
+         * Filter for Element, will include only known event type
          */
-        RiotFilter
+        ElementFilter
```
Comment (ganfra) [inline matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/sync/FilterService.kt:27]: should we not remove this from SDK?  and all stuff related to Element filter ofc

**Review by ganfra — APPROVED** [review]
One remark, it can be done later, but I guess it would be better, as it's a bit weird now

File: matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/sync/FilterService.kt (hunk 1 above)
Comment (bmarty) [reply matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/sync/FilterService.kt:27]: Yes, probably.

**Comment by bmarty** [comment]
Thanks @ganfra for the review

**Pull request merged**